doc/api.md

```diff
@@ -567,6 +568,28 @@ which is `details.internal_name` and falls back to `title`.
 - `document_type` *(required)*
   - The `document_type` value that returned editions has.
 
+## `GET /v2/links/changes`
+
+Returns an array of changes to links, giving the information on what
+changed (source and target content ids, and the link type), when the
+change happened, what user was associated with the action, and if the
+link was created or deleted.
+
+The results will be in decending order by link change id, which
```

Comment:
I wonder if created_at might be a more intuitive field to order by? I have a comment to make about link change id that is related that I'll add in a different spot.

Reply:
Done.